packages/sakuli-legacy/src/context/common/actions/screen.function.ts

```diff
@@ -37,7 +37,7 @@ export const getTimestamp = (
   return timestampString.split(":").join("-");
 };
 
-export type SearchResult = {
+export type ScreenRegionResult = {
```

Comment:
If a user specified the type of a variable explicitly, this would be a breaking change.

```typescript
let region: SearchRegion
try{
  region = await screen.find(...)
}
```